[core]/es_extended/server/functions.lua

```diff
@@ -259,6 +259,27 @@ function ESX.GetExtendedPlayers(key, val)
 	return xPlayers
 end
 
+function ESX.GetNumPlayers(key, val)
+    if(not key) then
+        return ESX.Table.SizeOf(ESX.Players)
```

Comment:
why not just `#ESX.Players` most likely more efficient correct me if i'm wrong

Reply:
The table ESX.Players contains nil value keys (most noteably when players drop from the server), so we have to use the nil proof alternative `ESX.Table.SizeOf(t)`

Reply:
Odd. I would've expected them to be removed on disconnect. probably there for some reason i'm unsure of but yeah I get what you mean now, apologies.

Reply:
> Odd. I would've expected them to be removed on disconnect.

They are? Hence the nil values.
It's a sparse array, so the length operator doesn't work.